tools/who_what_benchmark/whowhatbench/model_loaders.py

```diff
@@ -31,14 +31,41 @@ def __getattr__(self, attr):
             return getattr(self.model, attr)
 
 
-def load_text_genai_pipeline(model_dir, device="CPU", ov_config=None):
+def get_scheduler_config_genai(cb_config):
+    import openvino_genai
+
+    default_cb_config = {"cache_size": 1}
+    scheduler_config = openvino_genai.SchedulerConfig()
+    scheduler_params = cb_config or default_cb_config
+    if scheduler_params:
+        logger.info(f"Scheduler parameters for:\n{scheduler_params}")
+        for param, value in scheduler_params.items():
+            if param == "cache_eviction_config":
+                value = openvino_genai.CacheEvictionConfig(aggregation_mode=openvino_genai.AggregationMode.NORM_SUM, **value)
```

Comment:
aggregation_mode is not configurable via cb config?

Reply:
no, I don't see we need to make it configurable in the tools.